sweetest/src/main/java/com/mysugr/sweetest/framework/accessor/DelegatesAccessor.kt

```diff
@@ -32,13 +32,13 @@ class DelegatesAccessor(@PublishedApi internal val accessor: BaseAccessor) {
             }
 
             val dependencyState = TestEnvironment.dependencies.configurations.getAssignableTo(T::class)?.let {
-                TestEnvironment.dependencies.states[it]
-            } ?: kotlin.run {
-                TestEnvironment.dependencies.states[T::class]
-            } ?: kotlin.run {
-                accessor.dependencies.provide<T>()
-                requireNotNull(TestEnvironment.dependencies.states[T::class])
-            }
+                TestEnvironment.dependencies.states.getByConfiguration(it)
+            } ?: run {
+                TestEnvironment.dependencies.states.getByDependencyType(T::class)
+            } ?: error(
+                "No configuration or dependency state for ${T::class.simpleName} added. " +
```

Comment:
can you write an example what that means? e.g. "Add the dependency with "provide<$T::class.simpleName>" (or the deprecated way: to the module configuration)"